File: src/main/java/com/lord/small_box/dtos/DepositReceiverDto.java
package com.lord.small_box.dtos;

import java.util.Calendar;
import java.util.List;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DepositReceiverDto {

	private Long id;

	private Calendar receptionDate;

	private long organizationId;
	
	private String fromOrganizationName;

	List<DepositControlReceiverDto> depositControlReceivers;
	
	private String depositRequestCode;
	
	private boolean readed;

}

File: front/small-box/src/app/models/depositReceiverDto.ts
import { DepositControlReceiverDto } from "./depositControlReceiverDto";

export class DepositReceiverDto{
   id!:number;

	receptionDate!:Date;

	organizationId!:number;

	 depositControlReceivers!:DepositControlReceiverDto[];

	depositRequestCode!:string;

	readed!:boolean;
}